query-node/mappings/council.ts

```diff
@@ -0,0 +1,928 @@
+
+  const [memberId, stakingAccount, rewardAccount, balance] = new Council.NewCandidateEvent(event).params
+  const member = await getMembership(store, memberId.toString())
+
+  // specific event processing
+
+  // increase candidate count in stage update record
+  const lastStageUpdate = await getCurrentStageUpdate(store)
+  if (!isCouncilStageAnnouncing(lastStageUpdate.stage)) {
+    throw new Error(`Unexpected council stage "${lastStageUpdate.stage.isTypeOf}"`)
+  }
+
+  lastStageUpdate.stage.candidatesCount = new BN(lastStageUpdate.stage.candidatesCount).add(new BN(1))
+  await store.save<CouncilStageUpdate>(lastStageUpdate)
+
+  const electionRound = await getCurrentElectionRound(store)
+
+  // prepare note metadata record (empty until explicitily set via different extrinsic)
+  const noteMetadata = new CandidacyNoteMetadata({
+    bulletPoints: [],
+  })
+  await store.save<CandidacyNoteMetadata>(noteMetadata)
+
+  // save candidate record
+  const candidate = new Candidate({
+    stakingAccountId: stakingAccount.toString(),
+    rewardAccountId: rewardAccount.toString(),
+    member,
+
+    electionRound,
+    stake: balance,
+    stakeLocked: true,
+    candidacyWithdrawn: false,
+    votePower: new BN(0),
+    noteMetadata,
+    votesRecieved: [],
+  })
+  await store.save<Candidate>(candidate)
+
+  // common event processing - save
+
+  const newCandidateEvent = new NewCandidateEvent({
+    ...genericEventFields(event),
+    candidate,
+    stakingAccount: stakingAccount.toString(),
+    rewardAccount: rewardAccount.toString(),
+    balance,
+  })
+
+  await store.save<NewCandidateEvent>(newCandidateEvent)
+}
+
+/*
+  The event is emitted when the new council is elected. Sufficient members were elected and there is no other problem.
+*/
+export async function council_NewCouncilElected({ event, store }: EventContext & StoreContext): Promise<void> {
+  // common event processing - init
+
+  const [memberIds] = new Council.NewCouncilElectedEvent(event).params
+  const electedMemberIds = memberIds.map((item) => item.toString())
+  const members = await store.getMany(Membership, { where: { id: electedMemberIds } })
+
+  // specific event processing
+
+  // mark old council as resinged
+  const oldElectedCouncil = await getCurrentElectedCouncil(store, true)
+  if (oldElectedCouncil) {
+    oldElectedCouncil.isResigned = true
+    oldElectedCouncil.endedAtBlock = event.blockNumber
+    oldElectedCouncil.endedAtTime = new Date(event.blockTimestamp)
+    oldElectedCouncil.endedAtNetwork = CURRENT_NETWORK
+    await store.save<ElectedCouncil>(oldElectedCouncil)
+  }
+
+  // get election round and its candidates
+  const electionRound = await getCurrentElectionRound(store)
+
+  // TODO: uncomment when following query will be working (after some QN patches make it to Olympia)
+  ////const electedCandidates = await store.getMany(Candidate, { where: { electionRoundId: electionRound.id, member: { id_in: electedMemberIds } } })
```

Comment:
I'm not 100% sure if this will work, but did you try:
```typescript
import { In } from 'typeorm'
/* ... */
  store.getMany(Candidate, {
    where: { member: { id: In(memberIds.map((id) => id.toString())) } },
    relations: ['member'],
  })
/* ... */
```

Reply:
Loaded relations, unfortunately, don't have an effect on this. It's an error `"column worker.undefined does not exist"`  that we have encountered a couple of times lately, and it is not (fully?) integrated to Hydra yet. https://github.com/Joystream/hydra/issues/444